x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/security/authz/privilege/ClusterPrivilegeResolver.java

```diff
@@ -154,6 +155,11 @@
     public static final NamedClusterPrivilege CANCEL_TASK = new ActionClusterPrivilege("cancel_task",
         Set.of("cluster:admin/tasks/cancel"));
 
+    public static final NamedClusterPrivilege MANAGE_ENROLLMENT = new ActionClusterPrivilege("manage_enrollment",
```

Comment:
Pending the discussions with @albertzaharovits , if we leave the action name as is, then we should also change 
```
public static final NamedClusterPrivilege MANAGE_SECURITY = new ActionClusterPrivilege("manage_security", ALL_SECURITY_PATTERN,
            Set.of(DelegatePkiAuthenticationAction.NAME));
```
to
```
public static final NamedClusterPrivilege MANAGE_SECURITY = new ActionClusterPrivilege("manage_security", ALL_SECURITY_PATTERN,
            Set.of(DelegatePkiAuthenticationAction.NAME, "cluster:admin/xpack/security/enrollment*"));
```

and we should also restric